apps/mull-ui/src/app/pages/404/not-found.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const NotFoundPage = () => {
```

Comment:
```suggestion
const NotFound = () => {
```
Or change the file name to `not-found-page` so that it matches.

Reply:
I just put the name out of habit. 😅 There doesn't really seem to be a convention for the components in the pages folder.

Reply:
yeah that's unfortunately, its not a big deal but a fix ticket down the line could address this.